### PR TITLE
[Homework #4] - CSS by santiago pereira

### PR DESCRIPTION
**Pull request by Santiago-Pereira**

### CODE REVIEW

**Comment by Santiago-Pereira** [comment]
the last commit (d6d773b) is the one for homework 4 .
thank you!

File: homeworks/santiago-pereira/css-lecture-04/styles.css [hunk 1]
```diff
@@ -0,0 +1,306 @@
...
+    margin: 0 23px 30px 23px;
+    text-align: center;
+    font-size: 14px;
+    line-height: normal;
+    letter-spacing: -0.5px;
+}
+
+.main-content-buttons-selectors-portfolio{
+    width: 45%;
+    height: 50.303px;
```
Comment (ilya-ischenko) [inline homeworks/santiago-pereira/css-lecture-04/styles.css:177]: Where did you get such value for the height?

**Review by ilya-ischenko — COMMENTED** [review]
1. Fix merge conflict
2. Avatar should be 215x215px
3. Avatar should have white circle around it
4. Wrong font sizes for all texts
5. Add hover effect for "project card" as it showed on design
6. Add hover effects for buttons (change color, size, box shadow or smth else), you can use other website as an example
7. Check your code there - https://validator.w3.org/nu/#textarea and fix all warnings
8. Remove console logs from js code

**Comment by Santiago-Pereira** [comment]
I think all mistakes are fixed! lemme know if something is wrong.
I didn't make more animations cause I thought It would be nice to implement in homework 5.
thanks!

**Review by ilya-ischenko — COMMENTED** [review]
I see that you are trying to add hw3 and hw4 in this PR, fix it, please

**Pull request closed without merging**